decidim-design/app/views/layouts/decidim/design/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<% add_decidim_page_title("Decidim Design Guide") %>
+<% add_decidim_page_title(strip_tags(yield(:heading).lines.last).strip) if content_for?(:heading) %>
```

Comment:
Now this is: "Decidim Design Guide - Cards". 
This should be changed to the inverse ("Cards - Decidim Design Guide") so it's consistent with the other titles and layouts. It also makes more sense when you have multiple tabs and you can't read the full title. 
```suggestion
<% add_decidim_page_title(strip_tags(yield(:heading).lines.last).strip) if content_for?(:heading) %>
<% add_decidim_page_title("Decidim Design Guide") %>
```